pkg/controllers/user/nslabels/labels.go

```diff
@@ -0,0 +1,80 @@
+package nslabels
+
+import (
+	"reflect"
+	"strings"
+
+	"github.com/rancher/norman/clientbase"
+	typescorev1 "github.com/rancher/types/apis/core/v1"
+	"github.com/rancher/types/config"
+	"github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/meta"
+)
+
+const (
+	projectIDField = "field.cattle.io/projectId"
+)
+
+type namespaceHandler struct {
+	nsLister typescorev1.NamespaceLister
+	nsClient *clientbase.ObjectClient
```

Comment:
use the typed interface for the client `v3.NamespaceInterface`

Reply:
ok